packages/manager/core/generator/app/conditional-templates/listing/index.tsx.hbs

```diff
@@ -2,19 +2,23 @@ import React, { Suspense } from 'react';
 import { useTranslation } from 'react-i18next';
 import {
   Await,
-  defer,
   Link,
   Navigate,
   Outlet,
   useLoaderData,
 } from 'react-router-dom';
 
-import { {{listingEndpoint}} } from '@/api';
+import { {{listingEndpoint}}{{#if this.pimID}}, getPIMService, PIMElement{{/if}} } from '@/api';
 
 export function loader() {
-  return defer({
-    services: {{listingEndpoint}}(),
-  });
+  return Promise.all([{{listingEndpoint}}(){{#if this.pimID}}, getPIMService(){{/if}}]).then(
```

Comment:
we want to use useQuery, I have made a fix on this pr 
https://github.com/ovh/manager/pull/9447

Reply:
Done target switched to 'feat/MANAGER-11537' ✅